clients/admin-ui/src/features/common/nav/v2/nav-config.ts

```diff
@@ -57,34 +57,28 @@ export const NAV_CONFIG: NavConfigGroup[] = [
         title: "View map",
         path: "/datamap",
         requiresPlus: true,
-        scopes: [ScopeRegistryEnum.DATAMAP_READ],
+        scopes: [],
       },
       {
         title: "View systems",
         path: "/system",
-        scopes: [ScopeRegistryEnum.CLI_OBJECTS_READ],
+        scopes: [],
       },
       {
         title: "Add systems",
         path: "/add-systems",
-        scopes: [
-          ScopeRegistryEnum.CLI_OBJECTS_CREATE,
-          ScopeRegistryEnum.CLI_OBJECTS_UPDATE,
-        ],
+        scopes: [],
       },
       {
         title: "Manage datasets",
         path: "/dataset",
-        scopes: [
-          ScopeRegistryEnum.CLI_OBJECTS_CREATE,
-          ScopeRegistryEnum.CLI_OBJECTS_UPDATE,
-        ],
+        scopes: [ScopeRegistryEnum.DATASET_CREATE_OR_UPDATE],
```

Comment:
Even this one is misleading in the UI - that scope itself is enforced on that "ops"-style dataset endpoint but not the main "ctl"-style dataset endpoint.

So for the sake of avoiding any hiccups, let's do this instead:
```suggestion
        scopes: [],
```

Reply:
I did this 👍